### ISSUE TITLE
Link translation case (upper/lower/camel) 

### ISSUE THREAD

**Issue by dragantl** [kazupon/vue-i18n#106]
I find link translations a useful feature. However, it quickly became apparent that a bit of control over it would be great. For example:

```
{
   en: {
      home_address: 'Home address',
      missing_address: 'Please provide @:home_address'
   }
}
```

Using `en.missing_address` will result in `Please provide Home address`. It would be great to specify a case format for the link sting like so `Please provide @.lower:home_address`. Other modifiers can be used such as `upper` and `camel`. I think if more control is needed, formatting has to be used. Maybe this suggestion is also an overkill but seems like upper/lower/camel are most frequent modifiers.

**Comment by kissu** [comment]
Highly approved ! :dango:

**Comment by kazupon** [comment]
close due to implemented with #467
